server.js

```diff
@@ -1,10 +1,94 @@
-const express = require('express'); // remember to install your npm packages
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
As of version 4.16.0, Express has included a built-in JSON parsing middleware (https://expressjs.com/en/api.html#express.json). So we don't actually need `body-parser` anymore. We can just do this:

```
const express = require('express');
const server = express();

server.use(express.json());
```